frontend/src/components/App/index.tsx

```diff
@@ -69,22 +68,18 @@ class App extends Component<MyProps, MyState> {
   };
 
   render() {
-    const { message } = this.state;
 
     return (
       <StyledApp className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">{message}</h1>
-        </header>
+
```

Comment:
nit: Remove line? This is just a personal preference, as it seems readable if this is removed.